deflatebench.py

```
@@ -227,9 +215,26 @@ def benchmain():
     for level in map(str, getlevels()):
-        results[level] = []
+        result_comp[level] = []
+        result_decomp[level] = []
+
+    # Prepare compressed files for 'benchmode decomp only' mode
+    if benchmode == 'decomp' and cfgRuns['testmode'] != 'multi':
+        printnn("Compressing tempfiles for decompression test ")
+        if cfgRuns['testmode'] == 'single':
+            compfile = util.findfile(cfgSingle['testfile'])
+        else: # gen
+            compfile = util-findfile(cfgGen['srcFile'])
```

Ruff (0.14.14)

[error] Undefined name `findfile`

(F821)